Finilising configuring codeclimate

Fidesnoella opened the pull request:
**What does this PR do?**

- Configure test coverage by adding codeclimate ID to repo
- Add report generation
- Add badges to README

niyomozey committed:
setup app to support multiple languages

niyomozey committed:
Change on i18m configuration

Fidesnoella committed:
climate configuration

Fidesnoella committed:
climate configuration

Fidesnoella committed:
configure codeclimate

Fidesnoella committed:
configure codeclimate

cryptotearer committed:
Added configuration for sequelize with postgres for heroku

Fidesnoella committed:
Update README.md

Fidesnoella committed:
Add report generation

This add a script to generate the report and
send the report to code codeclimate in the workflow using
the code climate reporte ID

Fidesnoella committed:
Update README.md

niyomozey committed:
Changes added to support multiple language setup

Fidesnoella committed:
Merge branch 'develop' into continuous_integration

Fidesnoella committed:
Fixing testing issue

cryptotearer committed:
Edited some issues

cryptotearer committed:
Brought together all sequelize folders into the main folder called 'sequelize'

cryptotearer committed:
Brought together all sequelize folders into the main folder called 'sequelize'

Fidesnoella committed:
Remove file

Fidesnoella committed:
Update README.md

niyomozey committed:
Error resolved

Fidesnoella committed:
solving isue.

malaba6 reviewed (changes requested):
Why don't you rebase so this PR can be merged?

Fidesnoella committed:
Add report generation

This add a script to generate the report and
send the report to code codeclimate in the workflow using
the code climate reporte ID

Fidesnoella committed:
finish rebase

Fidesnoella committed:
setup app to support multiple languages

Fidesnoella committed:
Change on i18m configuration

Fidesnoella committed:
Changes added to support multiple language setup

Fidesnoella committed:
Add express structure and config (#9)

* add folder structure and missing configurations

* resolving testing issues

* removing unwanted scripts

* resolving @babel/runtime error

* fixing welcome message typos

* addition of extra structure with badge

* fixes es6 modules which prevents running

* fixing test error in addition.js

* fixing testing error

* resolving testing issues

* resolving @babel/runtime error

* addition of extra structure with badge

* fixes es6 modules which prevents running

* fixing testing error

* resolving issue with testing and code repetitions

* resolving issue with unwanted console and comments

* correcting misconfigurations inside folder structure

Fidesnoella committed:
fixing files inside structure to resolve conflicts

Fidesnoella committed:
configure codeclimate

Fidesnoella committed:
Update README.md

Fidesnoella committed:
setup app to support multiple languages

Fidesnoella committed:
Change on i18m configuration

Fidesnoella committed:
Changes added to support multiple language setup

Fidesnoella committed:
solving isue.

Fidesnoella committed:
finish rebase

Fidesnoella committed:
.

Fidesnoella committed:
Resolve test issue

Fidesnoella committed:
Resolve testing issues

Fidesnoella committed:
Resolving test issue

Fidesnoella commented:
> 

I have done with rebase you can merge

Fidesnoella commented:
> Why don't you rebase so this PR can be merged?
